tests/mock_network/churn.rs

```diff
@@ -707,13 +707,11 @@ impl Expectations {
                             *section_msgs_received.entry(key).or_insert(0usize) += 1;
                         }
                     } else {
+                        let dst_name = unwrap!(dst.as_node());
                         assert_eq!(
-                            orig_name,
-                            dst.name(),
+                            orig_name, *dst_name,
```

Comment:
Better to keep assert unique.
`assert_eq!( Ok(orig_name), dst_name, ...` instead of unwrap followed by assert.